scripts/test/lint.py

```diff
@@ -278,10 +278,12 @@ def check(self, path):
         
 class UsesInvalidCategory(Lint):
     # Some nuspec packages don't have a category
+    # excluded dcode because it fails to be installed
```

Comment:
[nitpick]  I think the comment should be in the same or previous line as the dcode exclusion. Please also add the issue URL and keep it consistent with the other exclusion.
```suggestion
```